Makefile

```diff
@@ -6,43 +6,30 @@ uninstall:
 clean:
 	rm -rf build dist
 	find . -name '*.pyc' -exec rm \{\} \;
+	find . -type d -name '__pycache__' -exec rm -r {} +
 
-deps:
-	pip install --upgrade pip build twine
-
-install: deps
+install:
 	@# Install OpenFisca-Extension-Template for development.
-	@# `make install` installs the editable version of OpenFisca-Extension-Template.
-	@# This allows contributors to test as they code.
-	pip install --editable .[dev] --upgrade
-
-build: clean deps
-	@# Install OpenFisca-Extension-Template for deployment and publishing.
-	@# `make build` allows us to be be sure tests are run against the packaged version
-	@# of OpenFisca-Extension-Template, the same we put in the hands of users and reusers.
-	python -m build
-	pip uninstall --yes openfisca-extension-template
-	find dist -name "*.whl" -exec pip install --force-reinstall {}[dev] \;
-
-check-syntax-errors:
-	python -m compileall -q .
+	@# The editable version of OpenFisca-Extension-Template allows contributors
+	@# to test as they code.
+	pip install --upgrade pip
```

Comment:
Do we still need to do this ?
```suggestion
```

Maybe it's because `tox` still use `pip` ?

Reply:
Yeah but tox is meant to simulate a new user installing things, so it shouldn't use make at all.